Examples/KleinGordon/StateVariables.hpp

```diff
@@ -0,0 +1,31 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifndef STATEVARIABLES_HPP
+#define STATEVARIABLES_HPP
+
+#include "ArrayTools.hpp"
+#include "BCParity.hpp"
+
+// assign an enum to each variable
+enum
+{
+    c_phi,
+    c_Pi,
+
+    NUM_VARS,
+};
+
+namespace StateVariables
+{
+static const amrex::Vector<std::string> names{"phi", "Pi"};
+
+static const std::array<BCParity, NUM_VARS> parities = {BCParity::even,
+                                                        BCParity::even};
+// The parities aren't really used, I just had to fill something in
```

Comment:
I think the wave solution is spherically symmetric so reflective BCs could be
used? I think the parities should be
```suggestion
static const std::array<BCParity, NUM_VARS> parities = {BCParity::even,
                                                        BCParity::odd};
```

Reply:
Actually, I think I'm wrong here. I got confused between spatial derivatives (which changes the parity) and time derivatives (which don't alter the parity). Could you change it back? Sorry about that :slightly_frowning_face: